src/sonic-config-engine/tests/simple-sample-graph.xml

```diff
@@ -180,11 +180,6 @@
           <AttachTo>fortyGigE0/4</AttachTo>
           <SubInterface/>
         </PortChannel>
-        <PortChannel>
-          <Name>PortChannel1001</Name>
-          <AttachTo>fortyGigE0/1;fortyGigE0/2</AttachTo>
```

Comment:
>fortyGigE0/1;fortyGigE0/2 [](http://example.com/codeflow?start=20&length=25)

They are existing ports. Why remove the portchannel?

Reply:
They are not existing, port name should be: fortyGigE0/0, fortyGigE0/4, fortyGigE0/8...
And this portchannel will not be displayed in config db for invalid member. if I replace member with valid port, config db output will be different, then I need to update test case.
So removing the portchannel seems to be a simple fix.